packages/Main/src/Layer/TiledGeometryLayer.js

```diff
@@ -430,18 +430,13 @@ class TiledGeometryLayer extends GeometryLayer {
             // TODO: pendingSubdivision mechanism is fragile, get rid of it
             node.pendingSubdivision = true;
 
-            const command = {
-                /* mandatory */
-                view: context.view,
-                requester: node,
-                layer: this,
-                priority: 10000,
-                /* specific params */
-                extentsSource: extents,
-                redraw: false,
-            };
-
-            return context.scheduler.execute(command).then((children) => {
+            const promises = [];
+
+            for (const extent of extents) {
+                promises.push(this.convert(node, extent));
```

Comment:
I think we can go further and remove the promises chain of this.convert since they are not created synchronously. I'll give it a try

Reply:
I'll do it in another PR so we can merge this one :)

Reply:
see #2525